docs/CONFIG-PROPERTIES.md

```diff
@@ -39,6 +39,7 @@
 | memory.apps.ignore.check | boolean | false | Ignore memory usage check for Apps|
 | newlog.gzipfiles.ondisk.maxmegabytes | integer in Mbytes | 2048 | the quota for keepig newlog gzip files on device |
 | process.cloud-init.multipart | boolean | false | help VMs which do not handle mime multi-part themselves |
+| edgeview.authen.jwt | edgeview session jwt token | empty string(edgeview disabled) | format as standard JWT for websocket session for temp testing |
```

Comment:
temp=temperature or temp=temporary?

Since this is temporary maybe we can even skip documenting it. Or have a comment say (here and/or in code) that edgeview.authen.jwt will be removed once controllers send EdgeViewConfig?

Reply:
I'll add a comment for this detail.